modules/apps/27-interchain-accounts/controller/client/cli/tx.go

```diff
@@ -13,14 +13,12 @@ import (
 
 	"github.com/cosmos/ibc-go/v6/modules/apps/27-interchain-accounts/controller/types"
 	icatypes "github.com/cosmos/ibc-go/v6/modules/apps/27-interchain-accounts/types"
-	clienttypes "github.com/cosmos/ibc-go/v6/modules/core/02-client/types"
 )
 
 const (
 	// The controller chain channel version
-	flagVersion                = "version"
-	flagPacketTimeoutHeight    = "packet-timeout-height"
-	flagPacketTimeoutTimestamp = "packet-timeout-timestamp"
+	flagVersion                        = "version"
+	flagRelativePacketTimeoutTimestamp = "relative-packet-timeout"
```

Comment:
Nit: why not drop the `Timestamp` word? In the proto file is called just `relative_timeout` and `timestamp` has the connotation of a fixed date/time...

Reply:
it must have been accidentally included! I've removed it now